governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+# Maker Executive Aug 2 2023
+
+# [Executive Proposal] DSR Increase, Spark D3M Changes, Harbor Trade Credit (RWA004-A) Changes, New Silver (RWA002-A) Changes, Launch Project Funding, AVC Member Participation Rewards, Clydesdale Quarterly Surplus Return - August 2, 2023
+
+The Governance Facilitator(s), AmusingAxl, Sidestream and Arby have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Increase the Dai Savings Rate (DSR) by 4.81% from 3.19% to 8%.
```

Comment:
You probably don't need the full from by x from y to z here. Compare to official version. https://github.com/makerdao/community/blob/master/governance/votes/Executive%20Vote%20-%20August%202%2C%202023.md